src/platform/src/main/java/org/geoserver/platform/RenderingEngineStatus.java

```diff
@@ -0,0 +1,114 @@
+/* (c) 2016 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.platform;
+
+import java.util.Optional;
+import sun.java2d.pipe.RenderingEngine;
+
+/**
+ * @author Morgan Thompson - Boundless
+ */
+public class RenderingEngineStatus implements ModuleStatus {
+
+    private static final String UNKNOWN = "unknown";
+    
+    private String name;
+    private String engine;
+    private String provider;
+    private Optional<String> component;
+    private Optional<String> version;
+    private Optional<String> message;
+    private Optional<String> docs;
+
+    @SuppressWarnings("unchecked")
+    public RenderingEngineStatus() {
+
+        name = this.getName();
+        
+        Class<RenderingEngine> renderer;
+        try {
+            renderer = (Class<RenderingEngine>) sun.java2d.pipe.RenderingEngine.getInstance()
+                    .getClass();
+        } catch (Throwable e) {
+            engine = UNKNOWN;
+            provider = UNKNOWN;
+            return;
+        }
+        engine = renderer.getSimpleName();
+
+        Package pkg = renderer.getPackage();
+        if (pkg.getName().contains("marlin")) {
+            provider = "Marlin";
+        } else if (pkg.getName().contains("sun.dc")) {
+            provider = "OracleJDK";
+        } else if (pkg.getName().contains("sun.java2d")) {
+            provider = "OpenJDK";
+        } else {
+            provider = pkg.getName();
+        }
+        message = this.getMessage();
+    }
+
+    @Override
+    public String getModule() {
+        return "jvm";
+    }
+
+    @Override
+    public Optional<String> getComponent() {
+        return component.of("java2d");
+    }
+
+    @Override
+    public String getName() {
+        return "Rendering Engine";
+    }
+
+    @Override
+    public Optional<String> getVersion() {
+        if (provider.contains("OracleJDK")) {
+            return version.ofNullable(System.getProperty("java.version"));
```

Comment:
Why do you have `Optional` instance `version` if all you ever use it for is to call these static methods?